### PR TITLE
feat(core): make fetch channel aware per locale

### PR DESCRIPTION
**Pull request by jorgemoya**
## What/Why?
- Add channel <=> locale config example.
- Make `client.fetch` channel aware by fetching channel id assigned to locale.
- Fetch from default channel if no map is found. 

## Testing
TBD

### CODE REVIEW

File: core/channels.config.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  fr: DEFAULT_CHANNEL_ID,
+  it: DEFAULT_CHANNEL_ID,
+  ja: DEFAULT_CHANNEL_ID,
+  nl: DEFAULT_CHANNEL_ID,
+  no: DEFAULT_CHANNEL_ID,
+  pl: DEFAULT_CHANNEL_ID,
+  'pt-BR': DEFAULT_CHANNEL_ID,
+  pt: DEFAULT_CHANNEL_ID,
+  sv: DEFAULT_CHANNEL_ID,
+};
```
Comment (BC-krasnoshapka) [inline core/channels.config.ts:24]: @jorgemoya does it make sense to combine this with locales from i18n into one config to minimize number of places where they're hardcoded?

Comment (jorgemoya) [reply core/channels.config.ts]: I have updated this object to just include the overrides, if any. I have made some type changes to make sure the keys match any of our configured locales. With these changes I don't think it makes sense to merge together, what do you think?

Comment (BC-krasnoshapka) [reply core/channels.config.ts]: Great idea!

**Review by chanceaclark — APPROVED** [review]

**Comment by jorgemoya** [comment]
Not working as expected for static sites.

**Pull request closed without merging**